src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/CountOperationTest.java

```diff
@@ -23,10 +23,12 @@
 import java.util.List;
 import java.util.function.Supplier;
 import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 
 @QuarkusTest
+@Disabled
```

Comment:
Are we planning to re-enable this test at some point?

Reply:
Need to figure out how to mock AsyncResultSet., then can be re-enabled.